pkg/relay/ethereum/signers/hd/config.go

```diff
@@ -0,0 +1,24 @@
+package hd
+
+import (
+	fmt "fmt"
+	"math/big"
+
+	"github.com/datachainlab/ethereum-ibc-relay-chain/pkg/relay/ethereum"
+)
+
+var _ ethereum.SignerConfig = (*SignerConfig)(nil)
+
+func (c *SignerConfig) Validate() error {
```

Comment:
Can we further validate `Mnemonic` and `Path` format using `GetPrvKeyFromMnemonicAndHDWPath`?

Reply:
@bluele Thank you! I fixed it now.